### PR TITLE
new file: Python/other/merge_interval.py

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes I read it   

### Description

Take an empty list and then apply a condition set of the list the last element is greater, lesser, equal to the number or not. when it equal it will add to the new list and then and then print the non-overlapping  intervals 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
 
The issue is related to #5224

### CODE REVIEW

File: Python/other/merge_interval.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        else:
+          new_pair = [current_pair[0],current_pair[1]]
+          intervals_stack.append(new_pair)
+
+      else:
+        intervals_stack.append(pair)
+   # result             
+  return intervals_stack
+
+interval = [[1, 3], [2, 6], [8, 10], [15, 18]]
```
Comment (HarshCasper) [inline Python/other/merge_interval.py]: Use dynamic input

File: Python/other/merge_interval.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          intervals_stack.append(new_pair)
+
+      else:
+        intervals_stack.append(pair)
+   # result             
+  return intervals_stack
+
+interval = [[1, 3], [2, 6], [8, 10], [15, 18]]  
+print("Overlapping interval: ",interval)
+print("Non-overlapping intervals: ",merge(interval))
```
Comment (HarshCasper) [inline Python/other/merge_interval.py]: Enclose in `if __name__ == '__main__'` block

File: Python/other/merge_interval.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Time complexity : O(n^2) 
+Space complexity : O(n^2) 
+
+INPUT:-
+Overlapping interval:  [[1, 3], [2, 6], [8, 10], [15, 18]]
+
+OUTPUT:-
+Non-overlapping intervals:  [[1, 6], [8, 10], [15, 18]]
+ 
+"""
```
Comment (HarshCasper) [inline Python/other/merge_interval.py]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/other/merge_interval.py (hunk 3 above)
Comment (soumikbaithalu) [reply Python/other/merge_interval.py]: @HarshCasper done

File: Python/other/merge_interval.py (hunk 2 above)
Comment (soumikbaithalu) [reply Python/other/merge_interval.py]: @HarshCasper done

File: Python/other/merge_interval.py (hunk 1 above)
Comment (soumikbaithalu) [reply Python/other/merge_interval.py]: @HarshCasper done

**Review by HarshCasper — APPROVED** [review]

**Comment by soumikbaithalu** [comment]
@HarshCasper sir PR policy is changed?

**Pull request merged**